src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/FormattingOptions2.IndentStyle.cs

```diff
@@ -0,0 +1,19 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.CodeAnalysis.Formatting
+{
+    internal partial class FormattingOptions2
+    {
+        /// <summary>
+        /// For use in the shared CodeStyle layer.  Keep in syntax with IndentStyle.
```

Comment:
```suggestion
        /// For use in the shared CodeStyle layer.  Keep in sync with IndentStyle.
```

Reply:
Fixes in https://github.com/dotnet/roslyn/pull/60462.  thanks